### PR TITLE
serve: Bundle web UI endpoint and additional cleanup of static serve

### PR DESCRIPTION
**Pull request by jacobhaven**
Pretty UI for Bundle (from remote). This doesn't support just uploading/pasting certificates yet.

Cleanup of CSS and Javascript, prettyfying code and pages.

Demo up here: https://cfssl.jhaven.me/bundle.html

### CODE REVIEW

**Comment by grittygrease** [comment]
Good first step towards a comprehensive bundle UI.

**Comment by jacobhaven** [comment]
In general, it might make sense to have all the pages (Index, Scan, Bundle, SelfSign) to actually be served as a single HTML file to side-step the lack of templates to avoid duplicated header/etc. code. I'm not sure the CSS/Javascript trickery needed to make this happen in a nice way.

**Comment by jacobhaven** [comment]
Bundle specifically should have another form option of uploading/pasting a certificate (similar to the cloudflare.com UI) to be bundled.

**Comment by grittygrease** [comment]
Can you remove the ".html" from each of the subpages? It seems old fashioned.

**Comment by grittygrease** [comment]
@jacobhaven This seems to be failing in Travis, can you rebase?

**Comment by jacobhaven** [comment]
There was actually a bug in the test where the server wasn't getting shut down (causing havoc with later tests). Everything should be good now.

File: cli/serve/serve.go [hunk 1]
```diff
@@ -91,15 +94,24 @@ var v1Endpoints = map[string]func() (http.Handler, error){
...
+	}
+	return (&url.URL{Path: prefix + endpoint}).String()
+}
+
 // registerHandlers instantiates various handlers and associate them to corresponding endpoints.
 func registerHandlers() {
 	for path, getHandler := range v1Endpoints {
 		if path != "/" {
-			path = fmt.Sprintf("/api/v1/cfssl/%s", path)
+			path = "/api/v1/cfssl/" + path
```
Comment (kisom) [inline cli/serve/serve.go]: Should this use the `v1APIPath` function defined above?

Comment (jacobhaven) [inline cli/serve/serve.go]: Yes. I'll update that in some future PR...

**Pull request merged**